getting-started/kubernetes/self-managed-public-cloud/azu.md

```diff
@@ -0,0 +1,73 @@
+
+### Before you begin...
+
+- Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Install [Azure CLI tools](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+### How to
+
+There are many ways to install and manage Kubernetes in Azure. This guide shows how to use **aks-engine** to deploy a cluster with **Azure’s CNI plugin for networking** and **{{site.prodname}} for network policy enforcement**. The advantage of this approach is that pods are assigned IP addresses associated with Azure Network Interfaces on worker nodes. The IPs come from the VNET network pool and therefore do not require NAT to access resources outside the Kubernetes cluster. However, there are other options that may work better for your environment.
+
+- [aks-engine for Azure networking and Calico network policy](#aks-engine-for-azure-networking-and-calico-network-policy)
+- [Other options and tools](#other-options-and-tools)
+
+#### aks-engine for Azure networking and Calico network policy
+
+[Install aks-engine](https://github.com/Azure/aks-engine/blob/master/docs/tutorials/quickstart.md#install-aks-engine) on your workstation.
+
+Before deploying, customize your cluster definition to use {{site.prodname}} for network policy.  Add or modify the `kubernetesConfig` section to include the following (see the [aks-engine documentation](https://github.com/Azure/aks-engine/blob/master/docs/topics/clusterdefinitions.md#kubernetesconfig) for other Kubernetes configuration settings).
+
+```
+"kubernetesConfig": {
+   "networkPlugin": "azure",
+   "networkPolicy": "calico"
+ }
+```
+ 
+Or, start with this [example cluster definition](https://github.com/Azure/aks-engine/blob/master/examples/networkpolicy/kubernetes-calico-azure.json) with these value already set, and customize to meet your needs. 
+
+Then, [follow the ask-engine documentation to deploy your cluster](https://github.com/Azure/aks-engine/blob/master/docs/tutorials/deploy.md), passing your cluster definition to `aks-engine deploy` via the `-m` flag. 
+
+#### Other options and tools
+
+##### {{site.prodname}} networking 
+
+You can also deploy {{site.prodname}} for both networking and policy enforcement. In this mode, {{site.prodname}} uses a VXLAN-based overlay network that masks the IP addresses of the pods from the underlying Azure VNET. This can be useful in large deployments or when running multiple clusters and IP address space is a big concern.
+
+Unfortunately, aks-engine does not support this mode, so you must use a different tool chain to install and manage the cluster. Some options:
+
+- Use [Terraform](https://www.terraform.io/) to provision the Azure networks and VMs, then [kubeadm](https://docs.google.com/document/d/1CsOj3smZQHrmu0jbY7hg1ShXfcTswYm0Qj4BrFdw5qs/edit#heading=h.imdkqm7uo7st) to install the Kubernetes cluster.
```

Comment:
kubeadm link is pointing to the original doc --- it's a cross reference.  Should be `[kubeadm](#kubeadm)` I think

Reply:
All inline links now.